src/frontend/src/tables/settings/UserTable.tsx

```diff
@@ -12,7 +19,10 @@ import {
   DetailDrawerLink
 } from '../../components/nav/DetailDrawer';
 import { ApiEndpoints } from '../../enums/ApiEndpoints';
-import { openCreateApiForm, openDeleteApiForm } from '../../functions/forms';
+import {
+  useCreateApiFormModal,
+  useDeleteApiFormModal
+} from '../../hooks/UseForm';
```

Comment:
Converted from old `openCreateApiForm` and `openDeleteApiForm` to newer `useCreateApiFormModal` and `useDeleteApiFormModal`

I believe this was leftover from the improvements done in https://github.com/inventree/InvenTree/pull/5897

This fixes an issue with the create new user button and the create new group button having different behavior on the admin page.